keyboards/bop/keymaps/via/rules.mk

```diff
@@ -0,0 +1,19 @@
+/* Copyright 2021 Nathan Blais
+  * 
+  * This program is free software: you can redistribute it and/or modify 
+  * it under the terms of the GNU General Public License as published by 
+  * the Free Software Foundation, either version 2 of the License, or 
+  * (at your option) any later version. 
+  * 
+  * This program is distributed in the hope that it will be useful, 
+  * but WITHOUT ANY WARRANTY; without even the implied warranty of 
+  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the 
+  * GNU General Public License for more details. 
+  * 
+  * You should have received a copy of the GNU General Public License 
+  * along with this program.  If not, see <http://www.gnu.org/licenses/>. 
+  */ 
+
+VIA_ENABLE = yes
+LTO_ENABLE = yes
+UNICODE_ENABLE = yes
```

Comment:
VIA doesn't support unicode, as far as I'm aware.  So no real point of having it enabled here. 

```suggestion
```